### ISSUE TITLE
The comparison is "too smart"

### ISSUE THREAD

**Issue by juanlanus** [CharlesStover/reactn-vs-redux#1]
Since a long time now, hardware resources are much less important than human resources.  
Yes, is it good that the reactn bundle is 15% smaller.  
But the real thing is that thousands of developer hours will be saved by an easy to understand state management feature. Entire lifespans, I'd say.  
  
On the other hand, the performance-oriented discussions are remains of the old time when CPU time was expensive, like during the early mainframe years.  
We developers are very fond of performance discussions and premature optimizations.  
  
IMO selling reduxn should be (almost) totally focused in the ease to understand the API (the devs UI), mentioning the better performance as a non-central byproduct.

**Comment by CharlesStover** [comment]
I agree. That has always been the driving inspiration behind the development of ReactN. This repository as made because there wasn't enough adoption yet to have any data on development time saved -- just myself as a single anecdotal data point. A hurdle of adoption for some users was performance, which mattered on their app, so I created this to showcase the only real data I could collect at the time.

Since this, ReactN has exploded, and the commentary from developers as far as 'hours saved' can speak for itself.

While the developer experience is and always will be the primary drive behind ReactN's future, a baseline to say that adopting ReactN won't crash your app or quadruple your memory consumption at the very least needs to exist, and this repo aims to provide that proof.

Unfortunately, I don't own an organization large enough to assign ReactN to A/B projects for numerical data on hours and money saved. That would be a greater repo, but beyond the scope of my abilities right now.

**Comment by juanlanus** [comment]
My testimony:  
I'm learning React & Redux, through an extensive online course in Eduonix.  
I struggle, because I'm not as good at JS, and as an old dog (74) I don't learn new tricks with ease.  
The course got overly complicated when it reached the Redux + Thunk part.  
Although the instructor showed how to do it, to me it was very difficult to grok at a leven in which I can be productive and not blindly cut&paste code I won't fully understand for quite a while. It was taking me a really long time, weeks.  
I was suspecting that all that boilerplate code production should be automated, until I stumped upon Reactn.  
Now I'm confident I will be able to develop the site I envision.

**Comment by CharlesStover** [comment]
I'm elated to hear it! I hope it makes your development experience easier. If you run into any issues, [the GitHub repo for ReactN itself](https://github.com/CharlesStover/reactn/issues) is welcoming!

**Comment by juanlanus** [comment]
Hi Charles,
I'm a ReactN user with a question: Is it safe to run ReactN with the newer
React versions?
I checked the README and it doesn't say I can't but also it doesn't say I
can.
My React hardcore tech skills are not enough to find it out by myself I'm
more of an usability guy).
Thanks!
--
Juan

**Comment by CharlesStover** [comment]
It is safe! In fact, I'd say it's preferred!

As long as ReactN is being maintained, it will support the latest and greatest version of React.

If React publishes breaking changes that ReactN cannot support, ReactN will publish a new major version and similarly stop supporting that dropped functionality going forward.

That is to say, the latest version of React should always be compatible with the latest version of ReactN.
